anta/tests/routing/bgp.py

```diff
@@ -447,3 +448,71 @@ def test(self) -> None:
 
         if failures:
             self.result.is_failure(f"Failures: {list(failures.values())}")
+
+
+class VerifyBGPTimers(AntaTest):
+    """
+    Verifies if the BGP peers are configured with the correct hold and keep-alive timers in the specified VRF.
+    Expected results:
+        * success: The test will pass if the hold and keep-alive timers are correct for BGP peers in the specified VRF.
+        * failure: The test will fail if BGP peers are not found or hold and keep-alive timers are not correct in the specified VRF.
+    """
+
+    name = "VerifyBGPTimers"
+    description = "Verifies if the BGP peers are configured with the correct hold and keep alive timers in the specified VRF."
+    categories = ["routing", "bgp"]
+    commands = [AntaCommand(command="show bgp neighbors")]
+
+    class Input(AntaTest.Input):
+        """
+        Input parameters for the test.
+        """
+
+        bgp_peers: List[BgpPeers]
+        """List of BGP peers"""
+
+        class BgpPeers(BaseModel):
+            """
+            This class defines the details of a BGP peer.
+            """
+
+            peer: Union[IPv4Address, IPv6Address]
+            """IPv4/IPv6 BGP peer"""
+            vrf: str = "default"
+            """VRF context"""
+            hold_time: int
+            """BGP hold time in seconds"""
+            keep_alive_time: int
+            """BGP keep-alive time in seconds"""
+
+    @AntaTest.anta_test
+    def test(self) -> None:
+        failures: dict[str, Any] = {}
+
+        # Iterate over each bgp peer
+        for bgp_peer in self.inputs.bgp_peers:
+            peer_address = str(bgp_peer.peer)
+            vrf = bgp_peer.vrf
+            hold_time = bgp_peer.hold_time
+            keep_alive_time = bgp_peer.keep_alive_time
+
+            # Verify BGP peer
+            if not (bgp_output := get_value(self.instance_commands[0].json_output, f"vrfs..{vrf}..peerList", separator="..")):
+                failures[str(peer_address)] = {vrf: "Not configured"}
+                continue
+
+            bgp_index = create_index(bgp_output, "peerAddress")
```

Comment:
Can you check if the [get_item](https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/plugins/plugin_utils/utils/get_item.py) function in AVD does what you want? If so, let's copy the code in `anta.tools` so we can use it.